task_geo/data_sources/hdx_oxfd/audit.md

```diff
@@ -0,0 +1,28 @@
+# Government Measures Dataset
+
+## General information
+
+- **Description**: The **Oxford COVID-19 Government Response Tracker** (OxCGRT) Dataset, provides a systematic way to
+track the stringency of government responses to COVID-19 across countries and time. Using a novel index that combines
+various measures of government responses, the authors describe variation in government responses, explore whether rising
+ stringency of response affects the rate of infection, and identify correlates of more or less stringent responses.
+
+
+OxCGRT collects publicly available information on 13 indicators of government response (S1-S13). Nine of the indicators
+(S1-S7, S12 and S13) take policies such as school closures, travel bans, etc. are recorded on an ordinal scale;
+the remainder (S8-S11) are financial indicators such as fiscal or monetary measures.
+For more details about the data go to: https://www.bsg.ox.ac.uk/research/research-projects/oxford-covid-19-government-response-tracker
+
+- **Credits**: HDX, Blavatnik School of Government, University of Oxford
+- **Source**: https://data.humdata.org/dataset/oxford-covid-19-government-response-tracker
+
+## Columns in the refined dataset
+
+
+
+## Transformations applied
+
+- Removed columns with notes
+- Converted ISO3 country code to ISO2 country code
+- Renamed columns
```

Comment:
Explain which columns have been renamed to what

Reply:
done